test/new-e2e/utils/e2e/client/vm_client.go

```diff
@@ -16,31 +16,31 @@ import (
 	"golang.org/x/crypto/ssh"
 )
 
-type sshClient struct {
+type vmClient struct {
 	client *ssh.Client
```

Comment:
Is there any valid use case where having the inner sshClient as public could be helpful ? We can keep it private for now

Reply:
I don't think we need client to be public.